gcore/gdalmultidim_gltorthorectification.cpp

```diff
@@ -124,6 +127,32 @@ class GLTOrthoRectifiedArray final : public GDALPamMDArray
         newAr->m_poSRS.reset(oSRS.Clone());
         newAr->m_poSRS->SetDataAxisToSRSAxisMapping(
             {/*latIdx = */ 1, /* lonIdx = */ 2});
+        if (CPLTestBool(CSLFetchNameValueDef(papszOptions,
+                                             "USE_BAD_WAVELENGTHS", "YES")) &&
```

Comment:
Is this spelled correctly? Shouldn't it be 
```suggestion
                                             "USE_GOOD_WAVELENGTHS", "YES")) &&
```
according to https://github.com/OSGeo/gdal/pull/10347/files#diff-47bb821c4b705cacab4b4ad82536126da726d7e5c7dcd9ea04e64c7207131e81R8451

Reply:
fixed